tests/lib/tests.py

```diff
@@ -98,7 +98,7 @@ def get_logs_path(conn, dbtype):
         pids = [pid for pid in listdir('/proc') if pid.isdigit()]
         for pid in pids:
             try:
-                cmdline = open('/proc/' + pid + '/cmdline', 'rb').read()
+                cmdline = open('/proc/' + pid + '/cmdline', 'r').read()
```

Comment:
I don't know why this worked at Ubuntu 20.04 (Python 3.8), but it doesn't work at Rocky Linux (Python 3.6 by default). Anyway the file behaves as text, not binary, and this way it works for Python 2 on CentOS7 as well.